iphone/Maps/Classes/Components/MWMNavigationController.m

```diff
@@ -60,7 +60,10 @@ - (void)traitCollectionDidChange:(UITraitCollection *)previousTraitCollection
 {
   [super traitCollectionDidChange: previousTraitCollection];
   // Update the app theme when the device appearance is changing.
-  [MWMThemeManager invalidate];
+  if ((self.traitCollection.verticalSizeClass != previousTraitCollection.verticalSizeClass)
```

Comment:
This will break the userInterfaceStyle changings which come from traitCollection.userInterfceStyle when device change its appearance and Auto is set in settings...